bin/server/bootstrap-letsencrypt.sh

```diff
@@ -5,7 +5,7 @@
 
 # shellcheck disable=SC2128
 
-domains=(davidrunger.com www.davidrunger.com)
+domains=(davidrunger.com grafana.davidrunger.com www.davidrunger.com)
```

Comment:
(In Cloudflare, I just edited the `ww2` A record entry (which I had to work around a LetsEncrypt rate limit that I was bumping up against the last time that I tried to renew SSL certificates) to `grafana`.)